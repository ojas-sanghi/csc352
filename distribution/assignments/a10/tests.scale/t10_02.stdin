8375075552
2379553374
4463908035
2719157076
9587833937
9086297289
7287920187
2366992412
5112477780
4363966018
