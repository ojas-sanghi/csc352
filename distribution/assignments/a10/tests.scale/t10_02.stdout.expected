8375075552 (audit=1)
scaled by 1: 8375075552 (audit=1)
scaled by 2: 16750151104 (audit=1)
scaled by 3: 50250453312 (audit=1)
scaled by 4: 201001813248 (audit=1)
scaled by 5: 1005009066240 (audit=1)
scaled by 6: 6030054397440 (audit=1)
scaled by 7: 42210380782080 (audit=1)
scaled by 8: 337683046256640 (audit=1)
scaled by 9: 3039147416309760 (audit=1)
2379553374 (audit=1)
scaled by 1: 2379553374 (audit=1)
scaled by 2: 4759106748 (audit=1)
scaled by 3: 14277320244 (audit=1)
scaled by 4: 57109280976 (audit=1)
scaled by 5: 285546404880 (audit=1)
scaled by 6: 1713278429280 (audit=1)
scaled by 7: 11992949004960 (audit=1)
scaled by 8: 95943592039680 (audit=1)
scaled by 9: 863492328357120 (audit=1)
4463908035 (audit=1)
scaled by 1: 4463908035 (audit=1)
scaled by 2: 8927816070 (audit=1)
scaled by 3: 26783448210 (audit=1)
scaled by 4: 107133792840 (audit=1)
scaled by 5: 535668964200 (audit=1)
scaled by 6: 3214013785200 (audit=1)
scaled by 7: 22498096496400 (audit=1)
scaled by 8: 179984771971200 (audit=1)
scaled by 9: 1619862947740800 (audit=1)
2719157076 (audit=1)
scaled by 1: 2719157076 (audit=1)
scaled by 2: 5438314152 (audit=1)
scaled by 3: 16314942456 (audit=1)
scaled by 4: 65259769824 (audit=1)
scaled by 5: 326298849120 (audit=1)
scaled by 6: 1957793094720 (audit=1)
scaled by 7: 13704551663040 (audit=1)
scaled by 8: 109636413304320 (audit=1)
scaled by 9: 986727719738880 (audit=1)
9587833937 (audit=1)
scaled by 1: 9587833937 (audit=1)
scaled by 2: 19175667874 (audit=1)
scaled by 3: 57527003622 (audit=1)
scaled by 4: 230108014488 (audit=1)
scaled by 5: 1150540072440 (audit=1)
scaled by 6: 6903240434640 (audit=1)
scaled by 7: 48322683042480 (audit=1)
scaled by 8: 386581464339840 (audit=1)
scaled by 9: 3479233179058560 (audit=1)
9086297289 (audit=1)
scaled by 1: 9086297289 (audit=1)
scaled by 2: 18172594578 (audit=1)
scaled by 3: 54517783734 (audit=1)
scaled by 4: 218071134936 (audit=1)
scaled by 5: 1090355674680 (audit=1)
scaled by 6: 6542134048080 (audit=1)
scaled by 7: 45794938336560 (audit=1)
scaled by 8: 366359506692480 (audit=1)
scaled by 9: 3297235560232320 (audit=1)
7287920187 (audit=1)
scaled by 1: 7287920187 (audit=1)
scaled by 2: 14575840374 (audit=1)
scaled by 3: 43727521122 (audit=1)
scaled by 4: 174910084488 (audit=1)
scaled by 5: 874550422440 (audit=1)
scaled by 6: 5247302534640 (audit=1)
scaled by 7: 36731117742480 (audit=1)
scaled by 8: 293848941939840 (audit=1)
scaled by 9: 2644640477458560 (audit=1)
2366992412 (audit=1)
scaled by 1: 2366992412 (audit=1)
scaled by 2: 4733984824 (audit=1)
scaled by 3: 14201954472 (audit=1)
scaled by 4: 56807817888 (audit=1)
scaled by 5: 284039089440 (audit=1)
scaled by 6: 1704234536640 (audit=1)
scaled by 7: 11929641756480 (audit=1)
scaled by 8: 95437134051840 (audit=1)
scaled by 9: 858934206466560 (audit=1)
5112477780 (audit=1)
scaled by 1: 5112477780 (audit=1)
scaled by 2: 10224955560 (audit=1)
scaled by 3: 30674866680 (audit=1)
scaled by 4: 122699466720 (audit=1)
scaled by 5: 613497333600 (audit=1)
scaled by 6: 3680984001600 (audit=1)
scaled by 7: 25766888011200 (audit=1)
scaled by 8: 206135104089600 (audit=1)
scaled by 9: 1855215936806400 (audit=1)
4363966018 (audit=1)
scaled by 1: 4363966018 (audit=1)
scaled by 2: 8727932036 (audit=1)
scaled by 3: 26183796108 (audit=1)
scaled by 4: 104735184432 (audit=1)
scaled by 5: 523675922160 (audit=1)
scaled by 6: 3142055532960 (audit=1)
scaled by 7: 21994388730720 (audit=1)
scaled by 8: 175955109845760 (audit=1)
scaled by 9: 1583595988611840 (audit=1)
